packages/react-examples/.storybook/preview.js

```diff
@@ -1,38 +1,14 @@
-// @ts-check
 import * as React from 'react';
 import { initializeIcons } from '@fluentui/font-icons-mdl2';
 import { configure, addParameters, addDecorator } from '@storybook/react';
 import { withInfo } from '@storybook/addon-info';
 import { withPerformance } from 'storybook-addon-performance';
-import { withCompatThemeProvider, withFluentProvider, withKeytipLayer, withStrictMode } from '@fluentui/storybook';
+import { withFluentProvider, withKeytipLayer, withStrictMode } from '@fluentui/storybook';
 
+addDecorator(withInfo);
```

Comment:
according to types and docs this is the preferred usage when no args are provided to the factory